File: Ejercicios5/p5/p5.c
#include <stdio.h>

string bibble;

void cEsGenial() {
    bibble = "Es mi perro";  /* global variable */
    int bibble;
    bibble = "Es mi cachorro";  /* local variable */
    printf("%s\n", bibble);  /* print local variable */
}

int main() {
    cEsGenial();
    printf("%s\n", bibble);  /* print global variable */
}

File: Ejercicios5/p5/p5.java
public class p5 {
    int bibble;

    void JavaApesta() {
        bibble = "Es un perro";  // global variable
        int bibble;
        bibble = "Es un cachorro";  // local variable
        System.out.println(x);  // local
    }

    public static void main(String[] args) {
        (new p5()).run();
    }

    public void run() {
        JavaApesta();
        System.out.println(x);  // global
    }
}
